https://github.com/aabc/ipt-netflow/commit/66e4304101010108892376866334ec9317b427d8.patch
https://bugs.gentoo.org/813993

From: ABC <abc@openwall.com>
Date: Wed, 14 Jul 2021 15:39:42 +0300
Subject: [PATCH] Namespace sk_error_report

sk_error_report is introduced in v5.14-rc1: e3ae2365efc14 ("net: sock:
introduce sk_error_report").
--- a/ipt_NETFLOW.c
+++ b/ipt_NETFLOW.c
@@ -1943,7 +1943,7 @@ static struct ctl_path netflow_sysctl_path[] = {
 #endif /* CONFIG_SYSCTL */
 
 /* socket code */
-static void sk_error_report(struct sock *sk)
+static void netflow_sk_error_report(struct sock *sk)
 {
 	struct ipt_netflow_sock *usock;
 
@@ -1974,7 +1974,7 @@ static struct socket *usock_open_sock(struct ipt_netflow_sock *usock)
 	}
 	sock->sk->sk_allocation = GFP_ATOMIC;
 	sock->sk->sk_prot->unhash(sock->sk); /* hidden from input */
-	sock->sk->sk_error_report = &sk_error_report; /* clear ECONNREFUSED */
+	sock->sk->sk_error_report = &netflow_sk_error_report; /* clear ECONNREFUSED */
 	sock->sk->sk_user_data = usock;
 	sock->sk->sk_reuse = SK_CAN_REUSE;
 
